packages/muelu/src/Graph/Containers/MueLu_Aggregates_decl.hpp

```diff
@@ -46,18 +46,25 @@
 #ifndef MUELU_AGGREGATES_DECL_HPP
 #define MUELU_AGGREGATES_DECL_HPP
 
+#include "MueLu_ConfigDefs.hpp"
+
+#include <Kokkos_StaticCrsGraph.hpp>
+#include <Tpetra_KokkosCompat_ClassicNodeAPI_Wrapper.hpp>
+
+#include "MueLu_Aggregates_fwd.hpp"
+
 #include <Xpetra_Map_fwd.hpp>
 #include <Xpetra_Vector_fwd.hpp>
 #include <Xpetra_VectorFactory_fwd.hpp>
 #include <Xpetra_MultiVector_fwd.hpp>
 
-#include "MueLu_ConfigDefs.hpp"
 #include "MueLu_BaseClass.hpp"
-#include "MueLu_Aggregates_fwd.hpp"
 
-#include "MueLu_Graph_fwd.hpp"
+#include "MueLu_LWGraph_kokkos.hpp"
+#include "MueLu_Graph.hpp"
 #include "MueLu_GraphBase.hpp"
 #include "MueLu_IndexManager.hpp"
+#include "MueLu_IndexManager_kokkos.hpp"
```

Comment:
Do IndexManager and IndexManager_kokkos do the same thing? Is the IndexManager still used? If not, remove include.

Reply:
If they don't do the same thing, how does one use IndexManager?

Reply:
They don't, unfortunately. They're totally different objects because `IndexManager` is an abstract class with I believe three implemented children, and `IndexManager_kokkos` is not derived from it due to concerns about vtables on GPUs when used for the structured multigrid

Reply:
So this effectively kills off functionality offered by IndexManager then?

Reply:
This PR in particular simply allows storage of an RCP to either `IndexManager` or `IndexManager_kokkos` since previously they corresponded to `Aggregates` and `Aggregates_kokkos`, respectively. It doesn't change any of the functionality offered by either `IndexManager` class, and I think it would have appeared in testing if it removed any functionality.

Reply:
Ok, that was my question above. If you know that they do the same thing you can remove the non-Kokkos one.

Reply:
They don't do the same thing unfortunately, and merging them will probably be a slightly messy step in #11658.
We'll have to reconcile the following:
```
class IndexManager : public BaseClass
class GlobalLexicographicIndexManager : public IndexManager<LocalOrdinal, GlobalOrdinal, Node>
class LocalLexicographicIndexManager : public IndexManager<LocalOrdinal, GlobalOrdinal, Node>
class UncoupledIndexManager : public IndexManager<LocalOrdinal, GlobalOrdinal, Node>
class IndexManager_kokkos : public BaseClass
```